stream_alert_cli/manage_lambda/package.py

```diff
@@ -66,7 +53,7 @@ def create(self):
             exit(1)
 
         # Extract any precompiled third-party libs for this package
-        if not self._extract_precompiled_libs(temp_package_path):
+        if self.precompiled_libs and not self._extract_precompiled_libs(temp_package_path):
```

Comment:
👍

Reply:
This logic just migrated out of `_extract_precompiled_libs` for simplicity